challenges.tex

```diff
@@ -0,0 +1,530 @@
+This section covers four types of challenges, instead of just a single one.
+Each challenge corresponds to the production of a product with one of the
+available complexities (C0, C1, C2, C3) in the \ac{RCLL} using either
+one or two \ac{RS}.\\
+For complexities C1, C2 and C3 the accumulated cost for mounting the required
+rings must be equal to 1, 2 and 3, respectively.
+\textbf{Task:} Produce all posted orders.\\
+\begin{table}[!htb]
+ \centering
+ \begin{tabular}{l|l|l}
+  Machines & Challenge type & Scoring \\\hline
+  2 & C0 & 30 \\
+  3 & C1 & 50 \\
+  4 & C1 & 50 \\
+  3 & C2 & 70 \\
+  4 & C2 & 70 \\
+  3 & C3 & 100 \\
+  4 & C3 & 100 \\
+ \end{tabular}
+ \caption{CX Challenge}
+ \label{tab:challenge-cx}
+\end{table}
+
+\subsubsection{Exploration + C0 Challenge}\label{sec:challenge-combine-exp-c0}
+A simple production task on a field with unknown machine positions.
+The challenge is to produce a product of complexity C0 without receiving the
+machine positions at the start of the production phase, resembling a unified
+exploration and production phase.\\
+\textbf{Task:} Produce all posted orders.\\
+Beating this challenges yields 50 points.
+
+%\subsubsection{RefBox Simulation Challenge}\label{sec:challenge-simulation}
+%A competition on the agent level. The RefBox provides a set of actions
+%that can be executed by sending dedicated commands to the RefBox via protobuf.
+%Hence no actual robot is required to participate.
+%\textbf{Task:} Play a regular RCLL game through the RefBox simulation
+%interface.\\
+%Participating in this challenge yields points based on the achieved in-game
+%points, see \reftab{tab:challenge-simulation}.
+%Additionally, the team scoring the highest points overall gets awarded another
+%$10$ points
+%\begin{table}[!htb]
+% \centering
+% \begin{tabular}{l|l|l|l}
+%  Points & Scoring \\\hline
+%  $[0,50)$ & 0\\% chktex 9
+%  $[50,150)$ & 20\\% chktex 9
+%  $[150,250)$ & 40 \\% chktex 9
+%  $[250,\infty)$ & 60 \\% chktex 9
+% \end{tabular}
+% \caption{CX Challenge}
+% \label{tab:challenge-cx}
+%\end{table}
+
+
+\subsubsection{Markerless Detection Challenge}\label{sec:markerless}
+Image recognition challenge to classify different machine types.\\
+\textbf{Task:} Autonomously label the machines shown in a set of pictures,
+a picture may contain more than one machine, which all count towards the
+total number of machines to detect.\\
+As a preparation for this challenge, a data set will be supplied to
+all participants which may be used for training and testing purposes
+\footnote{\url{https://fh-aachen.sciebo.de/s/MRghypMQSMhTYaC}}.
+The evaluation set for the challenge consists of a set of separate images.
+\begin{table}[!htb]
+ \centering
+ \begin{tabularx}{\linewidth}{l|l|l|l}
+  \% Correctly Classified & \% Wrongly Classified & \% Not Classified
+  & Scoring \\\hline
+  $x$ & $y$ & $z$ & $(x-y)\cdot30$
+ \end{tabularx}
+ \caption{Machine Detection Challenge}
```

Comment:
where is the image set? In the rulebook repo?

Reply:
the link is right there:
```
\footnote{\url{https://fh-aachen.sciebo.de/s/MRghypMQSMhTYaC}}.
```